js/jquery/fade.js

```diff
@@ -0,0 +1,123 @@
+import $ from 'jquery'
+
+const NAME = 'fade'
+const VERSION = '1.0.0-alpha.2'
+const DATA_KEY = 'axa.fade'
+const EVENT_KEY = `.${DATA_KEY}`
+const DATA_API_KEY = '.data-api'
+const JQUERY_NO_CONFLICT = $.fn[NAME]
```

Comment:
you convinced me @AndyOGo, let's do it. what do you think about `$.axaFade()`, `$.axaDrawer()` and `$.axaSearch()`?